rusty_pizza_server/src/order_model/order.rs

```diff
@@ -169,6 +169,21 @@ impl Order {
             })
         }
     }
+
+    pub fn user_has_to_pay(self, user: u32) -> Option<Money> {
+        match self.meals.get(&user) {
+            None => None,
+            Some(user_meals) => Some(user_meals.calculate_total_price()),
+        }
```

Comment:
I'm perfectly fine with this code and just want to note, that `map` could be used here to make it a little less verbose:
```rust
self.meals.get(&user).map(|user_meals| user_meals.calculate_total_price())
```

Reply:
Thanks for the hint